docs/_includes/content.md

```diff
@@ -3355,8 +3355,33 @@ You can view the future development items list in the [github project issues](ht
 
 <a name="editor-support"></a>
 ## Editor Support
+
+### Vim
+
 * Vim: [vim-cargo-make](https://github.com/nastevens/vim-cargo-make)
 
+### VSCode
+
+For debugging purposes there are some example .vscode files located within the [docs/vscode-example](./docs/vscode-example) directory
+
+You may also need:
+
+  * A local install of LLVM (For the LLDB Debugger) installed and reachable on the path
+  * VSCode Extension - CodeLLDB
+  * VSCode Extension - "rust-analyser" (not the "rust" one)
+  * VSCode Extension - "Task Explorer"
+  * VSCode Extension - "crates"
+
+### Rust Format
+
+After making changes to the code, rustfmt can be used to automatically reformat the code within the repository
+```
+# To install
+rustup component add rustfmt
+# To run on the cargo project
+cargo fmt
+```
+
```

Comment:
i'll remove this one as cargo make comes with built in formatting task that auto installs it for you